packages/codegen/src/codegen.ts

```diff
@@ -115,7 +117,7 @@ export async function generate({
     }
     export class XataClient extends DatabaseClient<DatabaseSchema> {
       constructor(options?: BaseClientOptions) {
-        super({ databaseURL: "${databaseURL}", ...options}, links);
+        super({ databaseURL: "${databaseURL}", ...options}, links, tables);
```

Comment:
To be honest, I would either switch order here or make it an object. Haven't done it to not break compatibility with existing code-generated clients, but maybe it's worth. WDYT?

Reply:
I'd leave it like it is for now. It'd make more sense to change the order but it's not a big deal and personally I don't think it's worth it at the moment because we need to deal with backwards compatibliity, like you say.